src/redux/modules/spinner/spinner.js

```diff
@@ -0,0 +1,97 @@
+/* @flow */
+import debug from 'debug';
+import {
+  UPDATE_USER_REQUEST,
+  UPDATE_USER_SUCCESS,
+  UPDATE_USER_FAILURE,
+} from 'redux/modules/user/user-actions';
+import {
+  LOGIN_REQUEST,
+  LOGIN_SUCCESS,
+  LOGIN_FAILURE,
+  LOGOUT_REQUEST,
+  LOGOUT_SUCCESS,
+} from 'redux/modules/auth/auth-actions';
```

Comment:
I wasn't sure how best to implement this, but your suggestion sounds good (using `messageId` as the param to the new `showSpinner()` method).